src/ConnectedKubernetes/ConnectedKubernetes.Autorest/custom/New-AzConnectedKubernetes.ps1

```diff
@@ -534,6 +530,7 @@ function New-AzConnectedKubernetes {
         # }
         try {
             if ((-not ([string]::IsNullOrEmpty($ProxyCert))) -and (Test-Path $ProxyCert)) {
+                $ConfigurationSetting["proxy"]["proxy_cert"] = $ProxyCert
```

Comment:
why do we set this back? If this line is necessary, then we should keep line 496 - line 499.